graphcanvas/graph_view.py

```diff
@@ -115,9 +119,9 @@ def _layout_changed(self, new):
         self._canvas.style = new
 
     def _on_hover(self, label):
-        print "hovering over:", label
+        print(u"hovering over: {}".format(text(label)))
```

Comment:
Hmm. But I'm assuming that `label` is a string of some form here, with no justification for that assumption...  What _is_ the Python type of `label` at this point?